plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -132,6 +139,17 @@ func (plugin *T128GraphQL) checkConfig() error {
 		return fmt.Errorf("extract_fields is a required configuration field")
 	}
 
+	if plugin.Deadline.Duration != DefaultDeadline {
```

Comment:
I feel like we don't need to expose the deadline to the user - instead just let the user set the timeout and always calculate the deadline under the covers.